app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+
+  def create
+    user = User.find_by_email(params[:email])
+    # If the user exists AND the password entered is correct.
+    if user && user.authenticate(params[:password])
+      # Save the user id inside the browser cookie. This is how we keep the user
+      # logged in when they navigate around our website.
+      session[:user_id] = user.id
+      redirect_to '/'
+    else
+    # If user's login doesn't work, send them back to the login form.
+      redirect_to '/login'
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
```

Comment:
Usually `reset_session` or `session.clear` here to completely clear the session on logout.